Sources/Auth/Services/App/AppRespondSubscriber.swift

```diff
@@ -28,16 +28,23 @@ class AppRespondSubscriber {
 
     private func subscribeForResponse() {
         networkingInteractor.responsePublisher.sink { [unowned self] subscriptionPayload in
+            let response = subscriptionPayload.response
             guard
-                let requestId = subscriptionPayload.response.id,
+                let requestId = response.id,
                 let request = rpcHistory.get(recordId: requestId)?.request,
                 let requestParams = request.params, request.method == "wc_authRequest"
             else { return }
 
             networkingInteractor.unsubscribe(topic: subscriptionPayload.topic)
 
+            if let errorResponse = response.error,
```

Comment:
guard?

Reply:
if we have guard and response is not error the function would return here but we want to handle error and success response right?

Reply:
> We sending error code in reject method, and recover AuthError with code here. Right?

correct